src/components/v2/V2Project/V2FundingCycleSection/PayoutSplitsCard.tsx

```diff
@@ -101,7 +102,7 @@ export default function PayoutSplitsCard({
               currency={distributionLimitCurrency}
               totalValue={distributionLimit}
               projectOwnerAddress={projectOwnerAddress}
-              showSplitValues
+              showSplitValues={!distributionLimit?.eq(constants.MaxUint256)}
```

Comment:
**suggestion**: I added `MAX_DISTRIBUTION_LIMIT` as a constant. Could we use it everywhere instead of `constants.MaxUint256`?

Reply:
Oh yep nice ok